contracts/voting-verifier/src/contract.rs

```diff
@@ -224,6 +225,15 @@ mod test {
             .to_string()
             .parse()
             .unwrap(),
+            MessageIdFormat::Base58SolanaTxDigestAndEventIndex => {
+                Base58SolanaTxDigestAndEventIndex::new_from_b58_encoded_signature_and_index(
+                    id, index,
```

Comment:
why don't you just Keccak512 the id and then just construct like the other cases? I think this is the only place you need this constructor right?

Reply:
Applied this suggestion at dbb8e0726d5427e4f43c3a2ab1feb34feda38fd3 . Seems this part of the code is only used in tests. Is the intention to just fill the array for the test cases like [here](https://github.com/eloylp/axelar-amplifier-fork/blob/b61954f6f8e455c033987e46d6c8bddff7dd1068/contracts/voting-verifier/src/contract.rs#L831-L832) ? (no prod use)

Reply:
yeah its just to generate some test data